core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -415,21 +415,17 @@ class KafkaServer(val config: KafkaConfig, time: Time = SystemTime) extends Logg
 
   // Copy the subset of properties that are relevant to Logs
   // I'm listing out individual properties here since the names are slightly different in each Config class...
-  private def copyKafkaConfigToLog(serverProps: java.util.Map[String, Object]): java.util.Map[String, Object] = {
+  private def copyKafkaConfigToLog(kafkaConfig: KafkaConfig): java.util.Map[String, Object] = {
 
     val logProps = new util.HashMap[String, Object]()
-    val entryset = serverProps.entrySet.iterator
+    val entryset = kafkaConfig.originals.entrySet.iterator
```

Comment:
Agree.

originals was created to pass custom configuration for metric reporters, serializers, authorizers and other plugin components. None of these exist at the Log level.